### ISSUE TITLE
running "sirepo admin create-examples" fails on alpha

### ISSUE THREAD

**Issue by moellep** [radiasoft/sirepo#3475]
Running admin create-examples on alpha fails with a python exception:
```
[py3; sirepo]$ sirepo admin create-examples
...
  File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/sirepo/auth/__init__.py", line 150, in get_module
    return _METHOD_MODULES[name]
KeyError: 'guest'
```

**Comment by e-carlin** [comment]
`SIREPO_AUTH_METHODS='guest' admin create-examples` will fix for now

**Comment by moellep** [comment]
Fixed with #3481
